metrics/prometheus.go

```diff
@@ -0,0 +1,92 @@
+package metrics
+
+import (
+	"strings"
+	"sync/atomic"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func NewPrometheusRegistry(registry *prometheus.Registry, service string) Registry {
+	return &PrometheusRegistry{registry, service}
+}
+
+type PrometheusRegistry struct {
+	registry *prometheus.Registry
+	service  string
+}
+
+func (source *PrometheusRegistry) NewTimer(path ...string) Timer {
+	var histogramOpts = prometheus.HistogramOpts{Namespace: "moira", Subsystem: source.service, Name: strings.Join(path, "_")}
```

Comment:
Let's move `"moira"` to const as this string is used heavily in this file.

Reply:
It was done in the following commits.